test/common/definitions.spec.js

```diff
@@ -40,6 +40,8 @@ describe('schema definitions', () => {
         insuranceGroupCode: stringGenerate(31),
       },
       {},
+      { insurancePolicyNumber: ' ' },
+      { insuranceGroupCode: ' ' },
```

Comment:
this should be insurancePolicyHolderName and insuranceName

Reply:
Good catch. Thank you!